docs/GettingStarted.md

```diff
@@ -1169,6 +1171,29 @@ alternate_client = Demo::Echo::Service.rpc_stub_class.new(
 
 The integration will ensure that the `configured_interceptor` establishes a unique tracing setup for that client instance.
 
+### hanami
+
+The `hanami` integration will instrument routing, action and render for your hanami application. To enable the `hanami` instrumentation, it is recommended to auto instrument with
+
+```
+gem 'ddtrace', require: 'ddtrace/auto_instrument'
+```
+
+and create an initializer file in your `config/initializers` folder:
+
+```ruby
+# config/initializers/datadog.rb
+Datadog.configure do |c|
+  c.tracing.instrument :hanami, **options
+end
+```
```

Comment:
Should we clarify that this is optional? (Since the defaults are applied by auto_instrument?)